Source/Scene/PrimitivePipeline.js

```diff
@@ -59,6 +59,9 @@ define([
 
         if (toWorld) {
             for (i = 0; i < length; ++i) {
+                if (!defined(instances[i].geometry)) {
```

Comment:
Minor, but why use `continue` here?  Why not, just `if (defined(...` or even put the check in `transformToWorldCoordinates` since the semantics would still be fine just like `remove(undefined)` is OK with our collections.